.github/workflows/updater.yml

```diff
@@ -0,0 +1,41 @@
+---
+name: Updater
+on:
+  push:
+    branches: [ master ]
```

Comment:
In most other places, we have specifically tended towards long-form lists for this type of thing in our workflow files, even if it’s just a single item

I _think_ you can just list the branch name directly though if it’s only one branch, without needing the list structure.